### PR TITLE
Project : Catalogue Of My Things

### PR DESCRIPTION
**Pull request by sumairq**
# In this pull request we implemented the following requirements for all the objects: 
## Video Presentation 
https://www.loom.com/share/549b8c3b8fb24712983ce392cbbb81f7

- Each class from the UML class diagram should be created in a separate .rb file.
- All associations (1-to-many relationships) and aggregations (parent-child relationships) between classes should be implemented.
- All properties visible in the diagram should be defined and set up in the constructor method. Exception: properties for the 1-to-many relationships should NOT be set in the constructor method. Instead, they should have a custom setter method created.
- All methods visible in the diagram should be implemented
- Add unit tests for all implemented methods.
- Create a main.rb file that will serve as your console app entry-point
- All data should be preserved by saving collections in .json files
- Create a schema.sql file with tables that will be analogical to the structure of the classes in your app. As you cannot implement inheritance in the database tables 
- add all properties and associations from the parent Item class as table columns to all tables based on the child classes. Thanks to that it can be used to store data in the future.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,87 @@
...
+
+- Microverse
+
+## 📝 License
+
+This project is [MIT](LICENSE) licensed.
+
+## Show your support
+
+Give a ⭐️ if you like this project!
```
Comment (emmanuellekamwa) [inline README.md:87]: - [ ] Hello. Hope you guys are fine. Please you have forgotten to add a link to your video presentation. Please try and add it.

**Review by emmanuellekamwa — CHANGES_REQUESTED** [review]
**STATUS: CHANGES REQUESTED 🛑⛔**
Hello @sumairq and @mukhammadrizooff 👋
Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- You documented your work professionally.✔
- Github inline comments feature is used properly.
- You asked for the changes in a polite and friendly way.

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (sumairq) [reply README.md:87]: Thank you for your feedback. We added the video link to the presentation just now.

**Review by Emmyn5600 — APPROVED** [review]
Hi @sumairq and @mukhammadrizooff 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

nodding-yes

## Optional suggestions

- [x] N/A.

##### _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

##### Cheers and Happy coding!:clap::clap::clap:
##### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**